client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/task-helper.ts

```diff
@@ -55,12 +56,21 @@ export function getEnhancedTasks(
 				case 'plan_selected':
 					taskData = {
 						title: translate( 'Choose a Plan' ),
+						subtitle: displayGlobalStylesWarning
+							? translate(
+									'Upgrade your plan to publish your colour changes and unlock tons of other features'
```

Comment:
I think we default to American English:
```suggestion
									'Upgrade your plan to publish your color changes and unlock tons of other features'
```

Reply:
oops, can't believe it's took me this long.